kube/bundle.yaml

```diff
@@ -176,6 +176,8 @@ data:
         bootstrappers:
             - filesystem
             - commitlog
+            - peers
+            - uninitialized
```

Comment:
could we call this something else? like `mark_uninitialized_success` -- or some such?

Reply:
Spoke offline and settled on uninitialized_topology